markdown_it/renderer.py

```diff
@@ -6,14 +6,33 @@ class Renderer
 rules if you create plugin and adds new token types.
 """
 import inspect
-from typing import MutableMapping, Optional, Sequence
+from typing import (
+    ClassVar,
+    MutableMapping,
+    Optional,
+    Sequence,
+)
 
 from .common.utils import unescapeAll, escapeHtml
 from .token import Token
 from .utils import OptionsDict
 
+try:
+    from typing import Protocol
+except ImportError:  # Python <3.8 doesn't have `Protocol` in the stdlib
+    from typing_extensions import Protocol  # type: ignore[misc]
 
-class RendererHTML:
+
+class RendererProtocol(Protocol):
+    __output__: ClassVar[str]
+
+    def render(
+        self, tokens: Sequence[Token], options: OptionsDict, env: MutableMapping
+    ) -> str:
```

Comment:
```suggestion
    ) -> Any:
```

for example `DocutilsRenderer.render` returns a `docutils.nodes.document`
(the `MarkdownIt.render` method also already returns `Any`)

Reply:
:+1: committed

Reply:
guess you need to also import Any